### PR TITLE
center and zoom fixed with tweaks

### PR DESCRIPTION
**Pull request by mridulnagpal**
#68

### CODE REVIEW

**Comment by mridulnagpal** [comment]
Apparently we can't set it initially at 0,0 as it gives an error saying latlng not defined for (NaN, NaN)

File: src/core/gridSystem.js [hunk 1]
```diff
@@ -70,6 +70,9 @@ module.exports = function gridSystem(options) {
                                if(lng.indexOf(".") != -1) lng = lng.split('.')[0] + '.' + lng.split('.')[1].slice(0,decimalPlacesAfterZero)
                                return '' + lng + 'W';
                            }
+                           else if(lng == 0) {
```
Comment (jywarren) [inline src/core/gridSystem.js]: what does this do? Maybe add a comment if it's not clear?

Comment (mridulnagpal) [inline src/core/gridSystem.js]: Actually where lng was 0 it did not return anything, so undefined was displayed, to correct that I added this, also we did not need N or S with it so it had to be separate.

**Comment by mridulnagpal** [comment]
I could choose 1,1 gives no error

**Comment by jywarren** [comment]
Hmm, why don't we try that. But why did you add '' only for lat and not lon?

**Comment by mridulnagpal** [comment]
Because for lat we return '' + lat anyways but in lng we return it in if statements only if none satisfies it returns undefined but for lat it has to return '' + lat, also setting it to 1,1

**Pull request merged**